frontend.py

```diff
@@ -24,6 +25,7 @@
 patterns = yaml.load(open('websites.yml'))
 
 sites = ' '.join(list(patterns['username_patterns'].keys()))
+css_rules = json.dumps(patterns['logos'])
```

Comment:
If it's a logo, then the name should be `logo` / `icon` Or `logoUrls` / `iconUrl`

Reply:
ya, good catch; this hasnt been updated since the yaml was slightly changed in a prior updated.